src/EditorFeatures/CSharpTest2/Recommendations/AssemblyKeywordRecommenderTests.cs

```diff
@@ -137,6 +146,17 @@ public async Task TestBeforeAssemblyAttribute()
 namespace Goo {}");
         }
 
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        [WorkItem(362, "https://github.com/dotnet/roslyn/issues/362")]
+        public async Task TestNotBeforeAssemblyAttributeWithoutOpenBrace()
+        {
+            await VerifyAbsenceAsync(
+@"
+$$
+[assembly: Whatever]
+namespace Goo {}");
+        }
```

Comment:
same case here.  add *another* test that has a using statement above this.  thanks!